src/config.rs

```diff
@@ -56,12 +60,17 @@ pub struct Options {
     #[serde(skip_deserializing, default)]
     pub disable_hygiene: bool,
 
+    #[cfg(target_arch = "wasm32")]
+    #[serde(default = "default_as_true")]
+    pub disable_hygiene: bool,
+
     #[serde(skip_deserializing, default)]
     pub disable_fixer: bool,
 
     #[serde(skip_deserializing, default)]
     pub global_mark: Option<Mark>,
 
+    #[cfg(not(target_arch = "wasm32"))]
```

Comment:
If I remember correctly, there's two version of `default_cwd `. One for wasm, one for non-wasm. Can you delete the wasm version?

Reply:
I can not find default_cwd for wasm anywhere. The only way to get the code to compile was to remove it.